schema/gen/src/lib.rs

```diff
@@ -40,6 +40,7 @@ pub fn build_schemas() -> MetaMap {
         QueryExecutionFailure,
 
         RegistrableBox,
+        UpgradableBox,
```

Comment:
should it be `Upgradeable`? @outoftardis

Reply:
both are valid